quota/mysqlqm/mysql_quota.go

```diff
@@ -124,3 +130,29 @@ func countFromTable(ctx context.Context, db *sql.DB) (int, error) {
 	}
 	return count, nil
 }
+
+// turnOffInformationSchemaCache turn off statistics caching for MySQL 8
+// To always retrieve the latest statistics directly from the storage engine and bypass cached values, set information_schema_stats_expiry to 0.
+// See https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_information_schema_stats_expiry
+func turnOffInformationSchemaCache(ctx context.Context, db *sql.DB) error {
+	opt := "information_schema_stats_expiry"
+	res := db.QueryRowContext(ctx, "SHOW VARIABLES LIKE '%"+opt+"%'")
+	var none, expiry string
+	err := res.Scan(&none, &expiry)
```

Comment:
Is it possible to `Scan` expiry directly into an `int` or will that fail if it's unset?
(Would changing the query to `SELECT FROM @@SESSION.information_schema_stats_expiry` help if so?)

Reply:
Thanks very much for your reminding. Indeed, “expiry” should be scanned into `int` type.

When this variable does not exist (versions before MySQL 8.0), `SELECT @@SESSION.information_schema_stats_expiry` statement will return error instead of empty resultset.